junit5-support/src/main/kotlin/in/specmatic/test/reports/OpenApiCoverageReportProcessor.kt

```diff
@@ -3,14 +3,21 @@ package `in`.specmatic.test.reports
 import `in`.specmatic.core.ReportConfiguration
 import `in`.specmatic.core.ReportFormatterType
 import `in`.specmatic.core.log.logger
-import `in`.specmatic.test.reports.coverage.OpenAPICoverageReport
+import `in`.specmatic.test.reports.coverage.console.OpenAPICoverageConsoleReport
+import `in`.specmatic.test.reports.coverage.json.OpenApiCoverageJsonReport
 import `in`.specmatic.test.reports.coverage.OpenApiCoverageReportInput
 import `in`.specmatic.test.reports.renderers.CoverageReportTextRenderer
 import `in`.specmatic.test.reports.renderers.ReportRenderer
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
 import org.assertj.core.api.Assertions.assertThat
+import java.io.File
 
 class OpenApiCoverageReportProcessor(private val openApiCoverageReportInput: OpenApiCoverageReportInput ): ReportProcessor {
```

Comment:
Minor comment, we can take care of it later also. Can we remove "Processor" from the names of the ReportProcessor and OpenAPICoverageReportProcessor?

Reply:
Or just move the saveAsJson method to openApiCoverageJsonReport.print()?